### DISCUSSION TITLE
Feature request: Granular PGP support

### DISCUSSION THREAD

**Discussion by C-Higgins** [simple-login/app#911] [Feature Requests]
Apologies if this is not the right place to write this.

## Current Behavior
- PGP encryption and signing is applied to a mailbox regardless of how the message arrives in the mailbox.
- Digital signing can only be enabled for the mailbox if encryption is also enabled; simplelogin including their digital signature requires me to supply my public key

## Desired behavior
- Allow the creation of PGP-enabled subdomains or directories, such that a mail sent to e.g. `anything@secure.aleeas.com` will apply PGP encryption to the mail, but a mail sent to other aliases will remain unencrypted. 
- Allow digital signing to be enabled independently from encryption. I would like the mail I receive to be signed by `signer@simplelogin.io` whether the mail is encrypted or not, since this has no relation to my own public key.

I realize the first bullet can be implemented already if the mailbox address supports aliasing, for example I can create a secure mailbox by using `myemail+secure@gmail.com` and have aliases sending to that mailbox. However, the second bullet cannot be worked around.

**Comment by nguyenkims** [comment]
@C-Higgins this is the right place to request a new feature :)
